suite/suite_test.go

```diff
@@ -32,18 +35,37 @@ func TestMain(t *testing.T) {
 		t.Fatal("options required")
 	}
 
-	testFn, there := suites[*testSuite]
+	config := gravity.LoadConfig(t, *configFile, *stateDir, *tag)
+
+	osFlavors := []string{}
+	for _, flavor := range strings.Split(*osFlag, ",") {
```

Comment:
shorter
```go
osFlavors := strings.Split(*osFlag, ",")
```
Off-topic: define a type of comma-separated string flags if more list parameters are to come:

```go
var osFlavors valueList
flag.Var(&osFlavors, "os", "ubuntu", "comma delimited list of OSes")

type valueList []string
func (r *valueList) String() string {return strings.Join(r, ",")}
func (r *valueList) Set(value string) error {
	*r = strings.Split(value, ",")
	return nil
}
```